src/modules/feedback/components/FeedbackBox.js

```diff
@@ -54,12 +64,17 @@ export class FeedbackBox extends React.PureComponent<PropsType> {
       isPositiveRatingSelected,
       onFeedbackOptionChanged,
       onCommentChanged,
-      onSubmit,
       comment,
       closeFeedbackModal,
-      theme
+      theme,
+      feedbackSent
     } = this.props
 
+    let errorMessage
+    if (['ERROR'].includes(feedbackSent)) {
```

Comment:
Why not `feedbackSent === 'ERROR'` ?

Reply:
Only to achieve equality with the code at Native-App.